blog/_posts/2021-11-29-tilt-news-november-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: "tilt-news-november-2021"
+date: 2021-11-29
+author: lian
+layout: blog
+title: "Tilt News, November 2021"
+image: "/assets/images/tilt-news-november-2021/pexels-frans-van-heerden-624015.jpg"
+image_caption: "Aurora Borealis behind an igloo. Photo by Frans Van Heerden from <a href='https://www.pexels.com/photo/aurora-borealis-624015/'>Pexels</a>"
+description: "introducing unsung heroes and another How-To video! 🎉"
+youtubeId: Wur9HPjZ2Jo
+tags:
+  - news
+---
+Bonjour Tilteurs et Tilteuses,
+
+December is just around the corner, which means it’s time for the end of the year madness again. Hard to believe 2021 is almost over, isn’t it? We’re planning a cool 2021 recap for the end of the year, so there’ll be no December newsletter 😢  
+But for now, the November edition comes packed with great reads, awesome features, and one or two glimpses behind the scenes!
+
+## Tilt Mentions
+We love to see more and more folks mentioning and discussing Tilt publicly! 
+
+* Clear Street’s Ottavio Hartman wrote a blog post on [leveraging Tilt for their local dev environment](https://medium.com/@ottopo/523045dd2ac8) dubbed “localkube”.
+* In “[Defining the Developer Experience](https://dzone.com/articles/developer-experience)”, Kevin Taylor, VP of Engineering at Caylent, ponders the implications of Cloud Native development on the evolution of DevX. Tilt is mentioned together with Skaffold as tools to help develop on Kubernetes.
+* In his book [Effortless Cloud-Native App Development Using Skaffold](https://books.google.com/books?id=vNJCEAAAQBAJ&pg=PA233&lpg=PA233&dq=effortless+cloud-native+app+development+using+skaffold+tilt&source=bl&ots=NUwNtLM0Ci&sig=ACfU3U38wdnpWJF78uxTiNKkbgYDSnHAhQ&hl=en&sa=X&ved=2ahUKEwjx1Z3jnYn0AhXPSt8KHVsCBnMQ6AF6BAgTEAM#v=onepage&q=effortless%20cloud-native%20app%20development%20using%20skaffold%20tilt&f=false), Ashish Choudhary introduces Tilt as a possible alternative to Skaffold, 
+
+Hopefully one day we’ll be able to add *your* take on Tilt too. Whether your experience with Tilt was superb or awful, we love to read about it! 🤓
+
+## Tilt News
+### Docs
+Long-time Tilters have probably already noticed that we have overhauled our [docs](http://docs.tilt.dev) and introduced a [new, shiny tutorial](https://docs.tilt.dev/tutorial/index.html).  
+So far, our docs were mainly a collection of information by maintainers. From now on, we are designing our docs with our users in mind first. Rather than giving you all the information you could possibly need (and more), we’re trying to understand which problems you are trying to solve and how to best guide you through the solution finding process.  
```

Comment:
```suggestion
So far, our docs were mainly a collection of information by maintainers. From now on, we are designing our docs with our users journeys first and foremost. Rather than giving you all the information you could possibly need, we’re trying to understand which problems you are trying to solve and how to best guide you through the solution finding process.  
```